COLLABORATOR_GUIDE.md

```diff
@@ -263,6 +263,24 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Introducing New Modules
+
+Semver-minor commits that introduce new core modules should be treated with
+extra care.
```

Comment:
I would add that they need to land with two CTC (or maybe TSC now) signoff, like any other breaking change.

Reply:
Would that be a should or a must?

Reply:
I think that's a must. It's serious business: we are adding new API that we will need to maintain to core. Removing those is hard, so the CTC should be notified to weight in.